lib/grn.h

```diff
@@ -135,6 +135,7 @@ typedef SOCKET grn_sock;
 
 # ifndef __GNUC__
 #  include <intrin.h>
+#  pragma intrinsic (_InterlockedExchangeAdd)
```

Comment:
Could you remove a space between `intrinsic` and `(`?

Reply:
I've fixed.